react-native-example/README.md

```diff
@@ -0,0 +1,130 @@
+# React Native Example
+
+This project shows how to use zipkin-js in a React Native context.
+
+This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app).
+
+Below you'll find information about performing common tasks. The most recent version of this guide is available [here](https://github.com/react-community/create-react-native-app/blob/master/react-native-scripts/template/README.md).
+
+## Updating to New Releases
```

Comment:
there are a lot of notes here.. are they copy/pasted from somewhere? If so, can we reference there instead.